### DISCUSSION TITLE
There seems to be some issue with this version juggluco8.1.1 of the server when extracting

### DISCUSSION THREAD

**Discussion by stephend61** [j-kaltes/Juggluco#186] [General]
Hi, I have been using this for a while now and it has been working perfectly up untill this latest version. 
I recentlty update to the latest jugguclo Server and noticed the extract doesn't seem to extract any data I just end up with an file with the headings only. It was working in 8.0.21 ok.
not sure what changed.

This is how I currently extract the data

../juggluco-server -M  -B /app/data_tmp.csv

Thanks for this great program.

**Comment by j-kaltes** [comment]
Yes, you are right. Juggluco server 8.1.5, should have solved the issue:
https://www.juggluco.nl/Juggluco/cmdline/index.html

**Comment by stephend61** [comment]
Thanks , that fixed the issue, sorry hadn't noticed the 8.1.5 version had been released.

**Comment by j-kaltes** [comment]
I created the update after your bug report. See https://github.com/j-kaltes/j-kaltes.github.io/commits/primary/
